openai-client/src/commonMain/kotlin/com.aallam.openai.client/OpenAIConfig.kt

```diff
@@ -22,4 +23,19 @@ public class OpenAIConfig(
     public val timeout: Timeout = Timeout(socket = 30.seconds),
     public val organization: String? = null,
     public val headers: Map<String, String> = emptyMap(),
+    public val host: OpenAIHost = OpenAIHost.OpenAI,
 )
+
+/**
+ * OpenAI host configuration.
+ */
+public class OpenAIHost(
+    /** Base URL configuration.*/
+    public val baseUrl: String,
```

Comment:
The actual implementation is not baseUrl but host and protocol.
If my baseUrl contains some path, this implementation will drop my path